kong/plugins/jwt/handler.lua

```diff
@@ -189,6 +189,12 @@ local function do_authentication(conf)
 
   set_consumer(consumer, jwt_secret, token)
 
+  for _, claim_name in ipairs(conf.inject_claims) do
+    if claims[claim_name] then
+      ngx_set_header("x-" .. claim_name, claims[claim_name])
```

Comment:
We should maybe prefix this header with something like `x-jwt` or `x-jwt-claim`. What do you think? Have you seen this done anywhere else, is it part of any standard you are aware of?

Reply:
I had a look into this now.
RFC 6648 of 2012 states that it discourages custom headers to be prefixed with x-.
Though, the set_consumer function in the same plugin is adding the consumer id, consumer custom id and consumer user name using the same x- prefix.
So if we go with the way that headers are currently added by the same plugin, then just x- should be used and not have the 'jwt' added. 

I'm happy to do it anyway you choose.